c2corg_ui/templates/base.html

```diff
@@ -55,7 +55,7 @@
     <button class="btn btn-default menu-open-close header"><span class="glyphicon glyphicon-align-justify"></span></button>
     <div class="page-header">
       <div class="quick-search">
-        <app-simple-search dataset="wr"></app-simple-search>
+        <app-simple-search dataset="wrc"></app-simple-search>
```

Comment:
"c" is "articles". See https://github.com/c2corg/v6_common/blob/master/c2corg_common/document_types.py#L9